ibm/service/power/resource_ibm_pi_cloud_connection.go

```diff
@@ -145,6 +145,11 @@ func ResourceIBMPICloudConnection() *schema.Resource {
 				Computed:    true,
 				Description: "GRE auto-assigned source IP address",
 			},
+			PICloudConnectionConnectionMode: {
```

Comment:
Could you please add this to `Attribute reference` in the resource doc?

Reply:
Will do. Thanks